infrastructure/modules/cloudfront/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_cloudfront_origin_access_control" "cloudfront_s3_oac" {
+  name                              = "CloudFront S3 OAC"
+  description                       = "Cloud Front S3 OAC"
+  origin_access_control_origin_type = "s3"
+  signing_behavior                  = "always"
+  signing_protocol                  = "sigv4"
+}
+
+resource "aws_cloudfront_distribution" "distribution" {
```

Comment:
Missing `logging_config` block

Reply:
are we prepared to create a new bucket for cloudfront logs ?